docs/swarm/join-nodes.md

```diff
@@ -29,6 +29,11 @@ also run tasks.
 Before you add nodes to a swarm you must install Docker Engine 1.12 or later on
 the host machine.
 
+The Docker Engine joins the swarm according to the **join-token** you provide to
```

Comment:
this is covered under #Join as a manager node in more detail:

When you run `docker swarm join` and pass the manager token, the Docker Engine
switches into swarm mode the same as for workers. Manager nodes also participate
in the raft consensus. The new nodes should be `Reachable`, but the existing
manager will remain the swarm `Leader`.

Docker recommends three or five manager nodes per cluster to implement high
availability. Because swarm mode manager nodes share data using Raft, there
must be an odd number of managers. The swarm can continue to function after as
long as a quorum of more than half of the manager nodes are available.

For more detail about swarm managers and administering a swarm cluster, see
[Administer and maintain a swarm of Docker Engines](admin_guide.md).